Sage/carbon
# feat: add feature flag to allow consumers to opt out of rounded corners update

### Proposed behaviour


Adds the `roundedCornersOptOut` prop to `CarbonProvider` to allow consumers to opt out of the rounded corner styling update

### Current behaviour


There is no opt out for consumers

### Checklist




#### QA


### Additional context



### Testing instructions


I will raise a PR against an older branch so that we can use chromatic as an initial review
The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.


https://codesandbox.io/s/thirsty-jasper-i0ecgb
